source/_integrations/mqtt.markdown

```diff
@@ -131,6 +119,31 @@ Note: This action does not remove the [MQTT broker](#setting-up-a-broker) or its
 
 {% enddetails %}
 
+<a name="configuration-via-subentries"></a>
+{% details "Configuration of MQTT components via Subentries" %}
+
+- [Notify](/integrations/notify.mqtt/)
+
+To add an MQTT device via a Subentry, follow these steps:
+
+1. Go to **{% my integrations title="Settings > Devices & services" %}**.
+2. Select the MQTT integration.
+3. Add a subentry via {% my integrations title="**Settings** > **Devices & services**" %}, click {% icon "mdi:dots-vertical" %} and select **Add MQTT device**.
+
+A device context and one or more entities can be added to the subentry.
+
+{% enddetails %}
+
+## Removing the integration
```

Comment:
Could the removal steps be moved to the end of the doc? or do you need them to be up here?

Reply:
We could move the instructions

Reply:
I would add a suggestion, but it's outside my powers here :sweat_smile:

Reply:
I'll push it

Reply:
done